### PR TITLE
Kl 2022 05 story issues

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

File: apps/home/blocks.py [hunk 1]
```diff
@@ -11,24 +11,6 @@ class ColorChoiceBlock(blocks.ChoiceBlock):
...
-    link = blocks.PageChooserBlock(required=False)
-    link_text = blocks.CharBlock(max_length=50, label='Link Text')
-    background_color = ColorChoiceBlock(
-        help_text='Not choosing a colour will result in a '
-                  'block with a white background.',
-        required=False
-    )
-
-    class Meta:
-        template = 'apps_home/blocks/cta_block.html'
```
Comment (phillimorland) [inline apps/home/blocks.py:28]: should we also remove template and any custom css?

**Review by phillimorland — COMMENTED** [review]
just one question otherwise looking good!

File: apps/home/blocks.py (hunk 1 above)
Comment (fuzzylogic2000) [reply apps/home/blocks.py:28]: yes, totally! Forgot about that and will do!

**Review by phillimorland — COMMENTED** [review]
perfect!

**Pull request merged**